src/sandstorm/update-tool.capnp

```diff
@@ -0,0 +1,54 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0x96c3fff3f4beb8fe;
+# This file contains schemas relevant to the Sandstorm self-updater.
+
+$import "/capnp/c++.capnp".namespace("sandstorm");
+
+struct PublicSigningKey {
+  key0 @0 :UInt64;
+  key1 @1 :UInt64;
+  key2 @2 :UInt64;
+  key3 @3 :UInt64;
+  # ed25519 public key.
+}
+
+struct Signature {
+  sig0 @0 :UInt64;
```

Comment:
Same thing as above - I guess this is a pattern when using Cap'n Proto for smallish bytearrays that have length of some multiple of 8 bytes?

This pattern seems convenient enough and highly performant for C++ implementations, but it's a pain when you're dealing with a higher-level language that might not expose the raw buffer to client code.  For example, when I was adding the `X-Real-IP` support, it was irritating to have to manually split a bytearray into 64-bit chunks, then encode those as hex strings (because Javascript lacks a 64-bit numeric type) to represent IP addresses, because `ip.capnp` represents the 128-bit IPv6 address as two 64-bit uints.  In practice, all the C library calls expect an array of 16 8-bit uints as declared as in <linux/in6.h>, and a typed array would be a more idiomatic representation in JS.  If we wind up ever using these in JS or some other language (say we reuse these structures for signing app updates, or something), or store these keys or signatures as JSONified-capnp in mongo, they get pretty annoying to work with.